src/Command/BasicCommands.php

```diff
@@ -128,4 +128,37 @@ public function xdebugStop()
             throw new \Exception("Failed, xdebug.ini not found.");
         }
     }
+
+  /**
+   * Create a database dump excluding devel and datastore tables.
+   *
+   * Run drush command on $alias to create a database dump excluding tables
+   * related to devel and datastore.
+   *
+   * @param String $alias Drush alias of the site we want the db from.
+   */
+  public function siteGrabDatabase($alias)
+  {
+    // Tables for which we want the structure and not the data.
+    $structure_tables_common = array(
+      'accesslog', 'batch', 'cache', 'cache_*', '*_cache', 'ctools_views_cache',
+      'ctools_object_cache', 'flood', 'history', 'queue', 'search_*',
+      'semaphore', 'sessions', 'watchdog'
+    );
+    $structure_tables_devel = array('devel_queries', 'devel_times');
+    $structure_tables_webform = array('webform_submitted_data');
+    $skip_tables = array('dkan_datastore_*');
+    $structure_tables = array_merge(
+      $structure_tables_common,
+      $structure_tables_devel,
+      $structure_tables_webform
+    );
+    $structure_tables_list = implode(', ', $structure_tables);
+    // Tables to be completely skipped.
+    $skip_tables = array('dkan_datastore_*');
```

Comment:
@dharizza do we need line 150? seems to be a duplicate of this one.

Reply:
You're right, fixed now.